docker-compose.yml

```diff
@@ -28,6 +29,7 @@ services:
     volumes:
       - ./../registry:/opt/webapps/app
       - ./shared/ca:/opt/ca
+      - tmp:/opt/webapps/app/tmp/
```

Comment:
@karlerikounapuu Is this related to cert generation or why is this new volume introduced ?

Reply:
Since registry now uses Rails 6.0, `bootsnap` came into action which speeds up app start from partial cache. As there are both `registry` and `epp` that use same `registry` image, cache may get into conflict because both containers try to access same cache in host. May end up with container start-up failures for few cycles when launching` docker-compose up -d`, unless it's present.